Jupyter Notebooks/LUVOIR/asahoo_merging_1.ipynb

```diff
@@ -0,0 +1,502 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eight-turtle",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pwd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "alike-tablet",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import LogNorm\n",
+    "from astropy.io import fits\n",
+    "import astropy.units as u\n",
+    "import hcipy as hc\n",
+    "import matplotlib as mpl\n",
+    "os.chdir('/Users/asahoo/repos/PASTIS/')\n",
+    "\n",
+    "import pastis.util as util\n",
+    "from pastis.config import CONFIG_PASTIS\n",
+    "\n",
+    "from pastis.e2e_simulators.luvoir_imaging import LuvoirAPLC\n",
+    "from pastis.e2e_simulators.luvoir_imaging import LuvoirA_APLC"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "occupational-texas",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ls"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "funded-romance",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nseg = 120            #for luvoir?\n",
+    "wvln = 638e-9\n",
+    "\n",
+    "datadir = '/Users/asahoo/laurent_repos/PASTIS/LUVOIR_delivery_May2019/' #confused what this directory actually does?\n",
+    "aper_path = 'inputs/TelAp_LUVOIR_gap_pad01_bw_ovsamp04_N1000.fits'\n",
+    "aper_ind_path = 'inputs/TelAp_LUVOIR_gap_pad01_bw_ovsamp04_N1000_indexed.fits'\n",
+    "aper_read = hc.read_fits(os.path.join(datadir, aper_path))\n",
+    "aper_ind_read = hc.read_fits(os.path.join(datadir, aper_ind_path))\n",
+    "\n",
+    "pupil_grid = hc.make_pupil_grid(dims=aper_ind_read.shape[0], diameter=15)\n",
+    "\n",
+    "aper = hc.Field(aper_read.ravel(), pupil_grid)\n",
+    "aper_ind = hc.Field(aper_ind_read.ravel(), pupil_grid)\n",
+    "\n",
+    "wf_aper = hc.Wavefront(aper, wvln)\n",
+    "hdr = fits.getheader(os.path.join(datadir, aper_ind_path))\n",
+    "\n",
+    "poslist = []\n",
+    "for i in range(nseg):\n",
+    "    segname = 'SEG' + str(i+1)\n",
+    "    xin = hdr[segname + '_X']\n",
+    "    yin = hdr[segname + '_Y']\n",
+    "    poslist.append((xin, yin))\n",
+    "    \n",
+    "poslist = np.transpose(np.array(poslist))\n",
+    "seg_pos = hc.CartesianGrid(hc.UnstructuredCoords(poslist))\n",
```

Comment:
You can delete this entire cell, this is internal to the simulator (this reminds me how old the branch Laurent was using really is...)